apps/etterlatte-saksbehandling-ui/server/src/utils/sanitize.ts

```diff
@@ -0,0 +1,6 @@
+export function sanitize(value: String | undefined) {
+  if (!value) {
+    return ''
```

Comment:
Så du ikke endrer oppførsel der vi sender inn undefined :D 
```suggestion
    return 'undefined'
```

Reply:
Nice 🎉